stdlib/Artifacts/src/Artifacts.jl

```diff
@@ -0,0 +1,634 @@
+module Artifacts
+
+import Base: get, SHA1
+using Base.BinaryPlatforms, Base.TOML
+
+export artifact_exists, artifact_path, remove_artifact, verify_artifact,
+       artifact_meta, artifact_hash, find_artifacts_toml, @artifact_str
+
+"""
+    parse_toml(path::String)
+
+Uses Base.TOML to parse a TOML file
```

Comment:
Feels like this doesn't really need a docstring? The reason to not add a dependency to the TOML stdlib is to keep this module as lean as possible? Could be added as a comment.

Reply:
Why would I use a comment instead of a docstring?  That seems like a strictly worse developer experience.  I'd much rather be able to look up this docstring in editors and the REPL.